astro/src/content/quickstarts/quickstart-go-api.mdx

```diff
@@ -0,0 +1,377 @@
+- `make-change`: This endpoint will allow you to call GET with a `total` amount and receive a response indicating how many nickels and pennies are needed to make change. Valid roles are `customer` and `teller`.
+- `panic`: Tellers may call this endpoint to call the police in case of an incident. The only valid role is `teller`.
+
+Both endpoints will be protected such that a valid JSON web token (JWT) will be required in the `Authorization` header in order to be accessed. Additionally, the JWT must have a `roles` claim containing the appropriate role to use the endpoint.
+
+<Aside type="note">
+If you simply want to run the application , there is a completed version in the 'complete-application' directory. You can use the following commands to get it up and running if you do not want to create your own.
+
+```shell
+cd complete-application
+go get
+go run main.go
+```
+
+You can then follow the instructions in the Run the API section started at Get a Token.
+</Aside>
+
+To create the application from scratch, make a directory for this API. 
+
+```
+mkdir your-application && cd your-application
+```
+
+### Initialize The Application
+
+Initialize the Golang application using the following:
+
+```
+go mod init your-application
+```
+
+### Write Golang Code
+
+<Aside type="note">
+This tutorial puts all these classes in the same package for simplicity, but for a production application, you'd separate these out.
+</Aside>
+
+Now you are going to write some Golang code. You need to create a file called `main.go`. Here you will add the package clause and import declarations below:
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/main.go"} tags="package-imports"
+  lang="go"/>
+
+You will also need to declare a variable and assign the value of the public key for the application. The value for this key will be set later in the application. Declare this variable below the import declarations.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/main.go"} tags="verify-key"
+  lang="go"/>
+
+### Add Handlers and Functions
+
+Create the main function for the application.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/main.go"} tags="main"
+  lang="go"/>
+
+Next, you are going to add some handlers. This tells the application how to handle specific requests. Notice the `isAuthorized` function wrapping around the `panic` and `makeChange` functions. We will implement this function later and it is what will validate the token and verify the roles. This function will also stand up the http listener on port `9001`.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/main.go"} tags="handlers"
+  lang="go"/>
+
+You will write two functions that will return values when users access them. These functions are called by the handlers. One is the `panic` which returns a message when a successful `POST` call is made to `/panic`. This will also display a message if the request is not made through a `POST`.
```

Comment:
```suggestion
You will write two functions that will return values when users access them. These functions are called by the handlers. One is the `panic` which returns a message when a successful `POST` call is made to `/panic`. This will also display a message if the request is not made through a `POST`. This API has no relation to the `panic` method in go, it's for a teller panicking when ChangeBank is being robbed.
```

Reply:
Didn't even know this was a thing.  Thanks!